pkg/webhook/mutation/pod/metadata/mutator.go

```diff
@@ -76,7 +76,7 @@ func (mut *Mutator) Mutate(ctx context.Context, request *dtwebhook.MutationReque
 
 	setupVolumes(request.Pod)
 	mutateUserContainers(request.BaseRequest)
-	updateInstallContainer(request.InstallContainer, workload)
+	updateInstallContainer(request.InstallContainer, workload, request.DynaKube.Status.KubernetesClusterMEID)
```

Comment:
It can happen that this field is empty. Setting an empty meid will lead to an init container crashloopbackoff, blocking the run of injected pods.

Reply:
Which scenario can lead to this to happen? We introduced the monitored entity reconciler exactly for the `KubernetesClusterMEID` field to be set.

Reply:
should be fixed now